File: nodejs/csv2json/person.js
export function processPerson(item) {
  return {
    id: Number(item.id),
    name: item.name,
    age: Number(item.age),
    height: Number(item.height),
    is_active: item.is_active === "true",
    created_at: new Date(item.created_at),
  };
}

File: python/csv2json/person.py
def process_person(item):
    return {
        "id": int(item["id"]),
        "name": item["name"],
        "age": int(item["age"]),
        "height": float(item["height"]),
        "is_active": item["is_active"] == "true",
        "created_at": item["created_at"]
    }
